e2e/cypress/integration/mailviewer.ts

```diff
@@ -5,11 +5,26 @@ describe('Interacting with mailviewer', () => {
         return cy.get('canvastable canvas:first-of-type');
     }
 
-    it('can reply to an email with no "To"', () => {
+    beforeEach(() => {
```

Comment:
So that we can close it if it pops up after all? It has no right to do that once we set the localstorage key – unless something breaks :)

Reply:
The conditional is for "if its open, close it, if its not open, carry on anyway" which is what the tests were stumbling on

Reply:
We have a test for it popping up here: https://github.com/runbox/runbox7/blob/master/e2e/cypress/integration/login.ts#L19

I'll extend it to check if it can be closed, and when it is, if it updates the localstorage like we expect. Then we'll be covered from both ends :)